src/Uno.UI.RuntimeTests/Tests/Windows_UI_Xaml_Controls/Given_TextBox.cs

```diff
@@ -305,5 +305,69 @@ public async Task When_IsEnabled_Set()
 			Assert.IsTrue(textbox.IsEnabled);
 			Assert.AreEqual(contentPresenter.Foreground, foregroundColor);
 		}
+
+#if __ANDROID__
+		[TestMethod]
+		public async Task When_Text_IsWrapping_Set()
+		{
+			var textbox = new TextBox();
+
+			var initialHeight = 500;
+
+			textbox.Height = initialHeight;
+
+			var stackPanel = new StackPanel()
+			{
+				Children = { textbox }
+			};
+
+			WindowHelper.WindowContent = stackPanel;
+
+			await WindowHelper.WaitForLoaded(textbox);		
+			
+			textbox.Text = "Sed ut perspiciatis unde omnis iste natus error sit voluptatem accusantium doloremaaaaaaaaaaaaaaaaaa";
+			textbox.TextWrapping = TextWrapping.Wrap;
+
+			Android.Graphics.Rect textBounds = new Android.Graphics.Rect();
+
+			textbox.GetHitRect(textBounds);
+
+			await WindowHelper.WaitForIdle();
+			
+			Assert.IsTrue(textBounds.Height() <= initialHeight);						
+		}
+
+
+		[TestMethod]
+		public async Task When_Text_IsNoWrap_Set()
+		{
+			var textbox = new TextBox();
+
+			var initialHeight = 250;
+
+			textbox.Height = initialHeight;
+
+			var stackPanel = new StackPanel()
+			{
+				Children = { textbox }
+			};
+
+			WindowHelper.WindowContent = stackPanel;
+
+			await WindowHelper.WaitForLoaded(textbox);
+
+			textbox.Text = "Sed ut perspiciatis unde omnis iste natus error sit voluptatem accusantium doloremaaaaaaaaaaaaaaaaaa";
+			textbox.TextWrapping = TextWrapping.NoWrap;
+
+			Android.Graphics.Rect textBounds = new Android.Graphics.Rect();
```

Comment:
This is likely not going to work on non-android platforms. Did `ActualHeight` not work?

Reply:
I saw a divergence about the Height measurement in-between TextBox.ActualHeight and the _ textBoxView (TextView wrapper) ActualHeigth, because of this my test fail. Can I equalize both values according to the Wrapper (TextView) values? Take a look at the image below, please

![image](https://user-images.githubusercontent.com/5515535/155215679-605e1ebe-4a81-4d19-93fe-8fdc89be2f05.png)

Reply:
This is the margins integrated in the TextBox. You can only validate that the height is bigger as wrapping is enabled.

Reply:
I did the change one more time and like I reported before ActualHeight never changes after wrapping and unwrapping the text. Still failing to assert condition.

![image](https://user-images.githubusercontent.com/5515535/155845446-38c036e5-3d23-488e-9be4-a8fd571598ca.png)